runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -335,4 +343,41 @@ public void onVarbitChanged(VarbitChanged event)
 			trawlerNotificationSent = false;
 		}
 	}
+
+   /**
+     * Changes the Fishing Trawler timer widget from minutes to minutes and seconds
+     * To the nearest 10 seconds
+     */
+    private void setTrawlerTimer() 
+    {
+		if (!config.trawlerTimer() || client.getGameState() != GameState.LOGGED_IN) {
+			return;
+		}
+		int regionID = client.getLocalPlayer().getWorldLocation().getRegionID();
+
+		if (regionID == TRAWLER_SHIP_REGION_NORMAL || regionID == TRAWLER_SHIP_REGION_SINKING) {
+			String trawlerText;
+		    Widget trawlerTimerWidget = client.getWidget(366, 37);
+
+		    //start's at 12 minutes
+            currentTimer = currentTimer.minusMillis(600);
+
+            trawlerText  = currentTimer.toString();
+            trawlerText  = trawlerText.replace("PT", "")
+                                     .replace("M", " Mins ")
+                                     .replace("S", " Secs");
+
+            //remove decimal
+            if (trawlerText.contains(".")){
+                trawlerText = trawlerText.replace(trawlerText.substring(trawlerText.indexOf("."), trawlerText.indexOf("S")), " ");
```

Comment:
All of this seems unnecessary. Just maintain an endtime and use an `.until(ENDTIME, ChronoUnits.SECODS)` and then use string formatter and set the text, not replace it.